.eslintrc.js

```diff
@@ -10,5 +10,7 @@ module.exports = {
     sourceType: "module",
   },
   plugins: ["react"],
-  rules: {},
+  rules: {
+    "react/prop-types": "off",
```

Comment:
Why have we set this property to `off`?

Reply:
Actually I setup eslint using "react" plugin. So it was asking me to define prop-types. Now next plugin is added. It is working fine. Thanks.